client/gutenberg/extensions/giphy/settings.js

```diff
@@ -0,0 +1,47 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { Path, SVG } from '@wordpress/components';
+
+export const settings = {
+	name: 'giphy',
+	icon: (
+		<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+			<Path fill="none" d="M0 0h24v24H0V0z" />
+			<Path d="M17.343 9.171h-4.114V5.057H7.057v13.886h10.286V9.17zm2.057-.514V21H5V3h10.286v2.057h2.057v2.057H19.4v1.543z" />
+		</SVG>
+	),
+	prefix: 'jetpack',
+	title: __( 'Giphy' ),
+	category: 'jetpack',
+	keywords: [ __( 'giphy' ) ],
+	description: __( 'Add a giphy graphic.' ),
+	attributes: {
+		align: {
+			type: 'string',
+			default: 'center',
+		},
+		caption: {
+			type: 'string',
+		},
+		giphyUrl: {
+			type: 'string',
+			default: '//giphy.com/embed/ZgTR3UQ9XAWDvqy9jv',
+		},
+		searchText: {
+			type: 'string',
+		},
+		topPadding: {
+			type: 'number',
+			default: 56.2,
```

Comment:
2 thoughts here:

- Just store the percent string we need in the style? (need to change the type)
- Did you attempt to parse this from the attribute? We're duplicating the ways this is stored. That's fine if it's complicated or fragile to get the value it out of the style attribute.

```suggestion
			default: '56.2%',
```

Reply:
Done in https://github.com/Automattic/wp-calypso/commit/aefd179d872ea0c58e0de15d435bd68fa65e6167 and https://github.com/Automattic/jetpack/commit/181de07ce813cf0cad08be143c41f33eac8b39e4